cmp/TK.apl

```diff
@@ -83,12 +83,12 @@ TK←{⍺←⊢
 	msk←3≤≢¨grp←(pos⊆⍨'⍺'=x),pos⊆⍨'⍵'=x
 	∨⌿msk:'AMBIGUOUS FORMALS'SIGNAL ∊msk⌿grp
 	msk←('⍺⍺'⍷x)∨'⍵⍵'⍷x ⋄ t[i←⍸msk]←P ⋄ end[i]+←1
-	t[⍸(~msk∨¯1⌽msk)∧x∊'⍺⍵']←A
+	t[⍸msk<(¯1⌽msk)<x∊'⍺⍵']←A
 
 	⍝ Tokenize primitives and atoms
-	t[⍸x∊syna]←A ⋄ t[⍸(~dm)∧x∊prms,'←']←P
-	msk←(x∊'⌶∇')∨msk∨¯1⌽msk←(~dm)∧⊃'⍠←' '∘←' '∘.'∨.⍷⊂x
-	end[⍸m2←2<⌿0⍪msk]←end⌿⍨2>⌿msk⍪0 ⋄ t[⍸msk∧~m2]←0
+	t[⍸x∊syna]←A ⋄ t[⍸dm<x∊prms,'←']←P
+	msk←(x∊'⌶∇')∨msk∨¯1⌽msk←dm<⊃'⍠←' '∘←' '∘.'∨.⍷⊂x
```

Comment:
The `(x∊'⌶∇')` check here looks odd. Semantically, it says that we want to capture an runs of `⌶` and `∇`, also extending currently masked digrams. _I.e._ it'll happily tokenize fun stuff like `⌶⍠←`, `⌶∇∇⌶`, `⌶∇∘.∇∇∇` _etc._